.buildkite/commands/setup_shared.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -eu
+
+echo "--- :ruby: Setting up Ruby tools"
+install_gems
```

Comment:
Decided to split between "shared" and "for building app" because, as you can see, they represent different steps. I trust that as the app grows in need, we'll make good use of the separation. But if nothing else, this saves us from duplicating the `echo "--- :ruby: Setting up Ruby tools"; install_gems` calls :)